engine/collection/compliance/engine.go

```diff
@@ -359,20 +359,25 @@ func (e *Engine) BroadcastProposalWithDelay(header *flow.Header, delay time.Dura
 	header.ChainID = parent.ChainID
 	header.Height = parent.Height + 1
 
-	log := e.log.With().
-		Hex("block_id", logging.ID(header.ID())).
-		Uint64("block_height", header.Height).
-		Logger()
-
-	log.Debug().Msg("preparing to broadcast proposal from hotstuff")
-
 	// retrieve the payload for the block
 	payload, err := e.payloads.ByBlockID(header.ID())
 	if err != nil {
 		return fmt.Errorf("could not get payload for block: %w", err)
 	}
 
-	log = log.With().Int("collection_size", payload.Collection.Len()).Logger()
+	log := e.log.With().
+		Str("chain_id", header.ChainID.String()).
```

Comment:
logging same fields as consensus compliance engine logging for main consensus blocks